kubespawner/proxy.py

```diff
@@ -189,18 +181,15 @@ async def add_route(self, routespec, target, data):
 
         async def ensure_object(create_func, patch_func, body, kind):
             try:
-                resp = await self.asynchronize(
-                    create_func, namespace=self.namespace, body=body
-                )
+                resp = (await create_func(namespace=self.namespace, body=body),)
```

Comment:
This is creating a tuple, which I assume is not intended. I don't think we use the response, though.

```suggestion
                resp = await create_func(namespace=self.namespace, body=body)
```

Reply:
Unintended.